src/main.rs

```diff
@@ -419,7 +419,8 @@ fn main() {
                         .long("threads")
                         .short("j")
                         .takes_value(true)
-                        .help("Set number of threads to use for searching"))
+                        .help("Set number of threads to use for searching\n\
+                               (default: The number of CPU cores on the system)"))
```

Comment:
Minor nitpick: I'd like to keep the width of the help text as small as possible. What do you think about `(default: number of available CPU cores)`?

Reply:
That's perfect, I'll update the PR.